src/main/java/core/basesyntax/ReadFromFile.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.IllegalFormatException;
+import java.util.IllegalFormatFlagsException;
+import java.util.List;
+
+public class ReadFromFile implements ReadableFile {
```

Comment:
I think this names aren't good. "ReadableFile" - sounds like file, that could be readed, not like interface. For interface you can take name "FileReader" or "CsvFileReader" etc.
"ReadFromFile" sounds like operation, it's name for method, not for class. Chouse "LocalFileReader", "FileReaderImpl" etc.

Reply:
done